src/components/feedback/banner/Banner.tsx

```diff
@@ -63,7 +66,11 @@ const BannerText = styled(Text)`
 	overflow: visible;
 `;
 
-const WrapAndGrowContainer = styled(Container).attrs(({ theme, gap, flexBasis }) => ({
+const WrapAndGrowContainer = styled(Container).attrs<{
+	theme: DefaultTheme;
+	gap: string;
+	flexBasis: ReturnType<typeof css> | string;
+}>(({ theme, gap, flexBasis }) => ({
 	flexBasis: css`calc(${flexBasis} + ${theme.sizes.icon.large} + ${gap})`
 }))``;
```

Comment:
The definition of the props is not needed, since all props already belong to the Container.
```suggestion
const WrapAndGrowContainer = styled(Container).attrs(({ theme, gap, flexBasis }) => ({
	flexBasis: `calc(${flexBasis} + ${theme.sizes.icon.large} + ${gap})`
}))``;
```